src/cobalt_model/config.rs

```diff
@@ -67,7 +67,7 @@ impl PageConfig {
             order: collection::SortOrder::None,
             rss: None,
             jsonfeed: None,
-            base_url: None,
+            base_url: site.base_url.clone(),
```

Comment:
I'm confused as to why the code didn't do this in the first place.  I'll need to dig into this

Reply:
Ok, this was previously needed only for RSS, so we just avoided sending along dead data.